core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage_ru.properties

```diff
@@ -0,0 +1,63 @@
+# The MIT License
+#
+# Copyright (c) 2022 CloudBees, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+Manage\ Old\ Data=Управление старыми данными
+Type=Тип
+Name=Имя
+Version=Версия
+Error=Ошибка
+Very\ old\ version=Очень старая версия
+Old\ Data\ Format=Старый формат данных
+Resave\ data\ files\ with\ structure\ changes\ no\ newer\ than\ Jenkins=Пересохранить файлы с изменениями в структуре \
+  данных не новее, чем текущая версия Jenkins
+No\ old\ data\ was\ found.=Старых данных не найдено.
+Unreadable\ Data=Нечитаемые данные
+Discard\ Unreadable\ Data=Удалить нечитаемые данные
+blurb.1=\
+  Когда происходят изменения в том, как данные хранятся на диске, Jenkins использует следующую стратегию: \
+  данные мигрируются в новую структуру во время загрузки, но записи не сохраняются заново в новом формате. \
+  Это позволяет откатить Jenkins к предыдущей версии, если это понадобится. Но таким образом данные могут остаться на \
+  диске в старом формате навсегда. В таблице ниже указаны записи, содержащие такие данные, и версии Jenkins, в \
+  которых структура данных была изменена.
+blurb.2=\
+  Иногда при чтении данных происходят ошибки (если отключить плагин, добавляющий какие-то данные, если \
+  не написан код миграции при изменении структуры данных, или если Jenkins откатить к предыдущей версии уже после \
+  того, как он записал данные, которые не может прочитать старая версия). \
+  Эти ошибки логируются, но после этого нечитаемые данные пропускаются, позволяя Jenkins запуститься и правильно \
+  работать.
+blurb.3=\
+  При помощи формы ниже можно привести эти записи в текущий формат. Если это сделать, при откате к старшей, чем \
```

Comment:
I lack a bit of context here tbh. My best shot is:
`При помощи формы ниже можно привести эти записи к новому формату. Если это сделать, то при откате к старшей версии Jenkins или плагина чтение данных сохраненных в новом формате станет невозможным.`

Reply:
the "чем выбранная" part is because it seems that you can choose a version to which you want to update - which is not necessarily the current, might be earlier? к новому формату does sound better, though.